calour/training.py

```diff
@@ -121,24 +126,202 @@ def split_train_test(exp: Experiment, field, test_size, train_size=None, stratif
     return train_X, test_X, train_y, test_y
 
 
-@Experiment._record_sig
-def collect_cv_prediction(exp: Experiment, field, estimator, cv, scoring, inplace=False):
-    '''Do the CV
+def classify(exp: Experiment, field, estimator, cv=RepeatedStratifiedKFold(3, 1),
+             predict='predict_proba', param_grid=None):
+    '''Evaluate classification during cross validation.
+
+    Parameters
+    ----------
+    field : str
+        column name in the sample metadata, which contains the classes we want to predict.
+    estimator : object
```

Comment:
maybe set sklearn.ensemble.RandomForestClassifer as the default value for the estimator? This way, people who don't know a lot about machine learning can use the classify function without needing to understand all the classifier choices (random forest is a good default for microbiome....)?

Reply:
nope. then they should read the docstring and probably shouldn't use it because this is a analysis requiring careful understanding